docs/marketplace/dev/front-end/table-design/table-features/table-feature-editable.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Table Feature Editable
+description: This article provides details about the Table Feature Editable component in the Components Library.
+template: concept-topic-template
+---
+
+This article provides details about the Table Feature Editable component in the Components Library.
+
+## Overview
+
+Table Feature Editable is a feature of the Table Component that allows editing/adding rows of the table.
+Below you can see base config of the Editable feature, that contains next options: 
+
+Feature Configuration:
+
+`columns` - an array with config for each editable column.  
+`create` - an object with config for a new rows.  
+`update` - an object with config for an existing rows.  
+`disableRowKey` - will disable row that contains mentioned column `id` (see example).  
+
+```html
+<spy-table [config]="{
+  dataSource: { ... },
+  columns: [ ... ],
+  editable: {
+    columns: [ ... ],
+    create: { ... },
+    update: { ... },
+    disableRowKey: 'col',
+  },                                                                                           
+}">
+</spy-table>
+```
+
+Let's take a closer look at all the possible options.
+
+`columns`:  
+  - `id` - a cell `id`.  
+  - `type` - a cell `type`.  
+  - `typeOptions`:  
+    - `value` - will set default value to the cell of the newly added row.  
```

Comment:
Please, mention that you can use every `typeOptions` variant depended on table column type (with link to it)